packages/sources/svelte/src/components/indicators/VtmnRating/VtmnRating.svelte

```diff
@@ -89,29 +95,12 @@
   };
 </script>
 
-<div class={componentClass} aria-disabled={disabled} {...$$restProps}>
-  {#if !readonly}
-    <div
-      class="vtmn-rating--interactive"
-      aria-label="Rate the article"
-      role="radiogroup"
-      data-rating={value}
-    >
-      {#each Array(starsCnt) as _, index}
-        {@const position = index + 1}
-        <input
-          type="radio"
-          bind:group={value}
-          {name}
-          value={position}
-          id={`${name}-${position}`}
-          aria-label={`${position} star out of 5`}
-          {disabled}
-        />
-        <label for={`${name}-${position}`} />
-      {/each}
-    </div>
-  {/if}
+<div
+  class={componentClass}
+  aria-disabled={disabled}
+  {...$$restProps}
```

Comment:
Thanks @Mig-l, this aria-label wasn't it already there in flat pass mode thanks to `{...$$restProps}`?

Reply:
@lauthieb , yes absolutely but i need this props in two places in the component.

Reply:
@Mig-l @lauthieb what do you think about this solution ? 
All the aria labels are grouped on 1 object.

```
export let ariaLabel = {};

<VtmnRating ariaLabel={{
  rateArticle: 'Rate the article',
  ratingStarPosition: '{0} star out of 5',
  ratingNumber: 'number of rating',
  articleRating: 'article rating'
}} />
```

Reply:
The @Tlahey' solution also makes the composant easier to maintain (for i18n for instance) and to use :) 
Thanks!

Reply:
Thanks @Mig-l @thibault-mahe & @Tlahey for your messages.
Yes completely agree, the solution proposed by @Tlahey is the best.

Reply:
Done